docs/european-user-consent.mdx

```diff
@@ -127,14 +131,57 @@ Once the user has selected their preference, the `showForm` method returns an `A
 ```js
 import { AdsConsent, AdsConsentStatus } from 'react-native-google-mobile-ads';
 
-const consentInfo = await AdsConsent.requestInfoUpdate();
+const requestAdsConsent = async () => {
+  const consentInfo = await AdsConsent.requestInfoUpdate({
+    tagForUnderAgeOfConsent: false,
+    testDeviceIdentifiers: [TEST_DEVICE_ID],
+    // Always ensure debug information is removed for production apps!
+    ...__DEV__ && { debugGeography: AdsConsentDebugGeography.EEA },
+  });
+
+  if (
+    consentInfo.isConsentFormAvailable &&
+    (consentInfo.status === AdsConsentStatus.REQUIRED ||
+      consentInfo.status === AdsConsentStatus.UNKNOWN)
```

Comment:
Wouldn't say `UNKNOWN` (->)[https://developers.google.com/admob/ump/ios/quick-start]

Reply:
@Gregoirevda can you expand on that? I'm unsure what you mean.
Do you mean that given the information in the link you provide as evidence, the AdsConsentStatus.UNKNOWN type will never be seen here? Implying that this clause of the conditional should be removed?

@VictorioMolina does that make sense as an analysis (assuming it is correct)?

Reply:
@mikehardy It depends on the degree of precision with which we want to ensure consent. The UNKNOWN status here is perfectly valid, and, although it is true that it could bother the user (the probability of this happening is really low) that the consent form is displayed every time the app is opened, but, with this, we make sure we comply with the law.

Reply:
@VictorioMolina showing a popup doesn't comply with the law. Sharing consent choices with the ad requests does.

Reply:
@Gregoirevda AFAIK AdMob handles the user choices automatically, based on the result obtained from the UMP form. Previously I thought that this was not the case, and that we had to manage whether or not to show personalized ads. But, according to the official google docs, the most restrictive decision always prevails, and there is no need to manage the permission on our part (as long as we use UMP).

Reply:
Thats why I think showing the UMP consent modal is a good choice when the status is unknown.